examples/se2_inverse.py

```diff
@@ -0,0 +1,65 @@
+# This example will use torch to solve min_x1 || x1^-1 - x2 ||^2,
+# where x1 and x2 are SE2 objects
+
+import torch
+import theseus as th
+from theseus import LieGroupTensor
+from theseus.geometry.lie_group import LieGroup
+
+from theseus.geometry.tests.test_se2 import create_random_se2
```

Comment:
We could move more generally useful stuff like create_random_se2 from tests to theseus/utils.

Reply:
Created #78 for this.